job/server/src/test/java/alluxio/job/plan/BatchedJobDefinitionTest.java

```diff
@@ -107,20 +101,16 @@ public class BatchedJobDefinitionTest {
 
   @Before
   public void before() throws Exception {
-    mMockFileSystem = PowerMockito.mock(FileSystem.class);
-    mMockBlockStore = PowerMockito.mock(BlockStoreClient.class);
-    mMockFsContext = PowerMockito.mock(FileSystemContext.class);
-    PowerMockito.mockStatic(BlockStoreClient.class);
```

Comment:
Same question... What happened to this static mock? Is it actually not necessary?